tutorials/android_studio/index.html

```diff
@@ -160,14 +159,15 @@ <h1 class="title">Processing for Android</h1>
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     FrameLayout frame = new FrameLayout(this);
-    frame.setId(ViewIdGenerator.getUniqueId());
```

Comment:
ViewIdGenerator handles the lack of generateViewId when API level is < 17:

https://github.com/processing/processing-android/blob/master/core/src/processing/android/ViewIdGenerator.java#L18